### PR TITLE
Change the way we manage placeholder images

### PR DESCRIPTION
**Pull request by cveneziani**
- Add avatar component
- Present images anywhere they are required
- Add user picture to profile edition

### CODE REVIEW

**Comment by nfilzi** [comment]
Reviewing this one tomorrow, to tired now.

**Comment by cveneziani** [comment]
Sure! 😃

File: apps/web/presenters/housing_presenter.rb [hunk 1]
```diff
@@ -5,6 +5,14 @@ def card_partial_name
     imported ? :housing_card : :housing_card_pending
   end
 
+  def picture
+    'housing-picture-placeholder.jpg'
+  end
+
+  def proposer_avatar
```
Comment (nfilzi) [inline apps/web/presenters/housing_presenter.rb]: `proposer`, like the organizer who proposed the housing?

Comment (cveneziani) [reply apps/web/presenters/housing_presenter.rb]: Well, yes.
It's the naming we've chosen back in May 😬 

See [here](https://github.com/nfilzi/housing-list/blob/master/apps/web/templates/housings/_housing_card.html.erb#L11-L17)

Shall we find a better one?

Comment (nfilzi) [reply apps/web/presenters/housing_presenter.rb]: nope, that's good then!

**Pull request merged**